forum/forum/routing.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+
+class Consumer(AsyncWebsocketConsumer):
+    async def connect(self):
+        await self.accept()
+        await self.send(text_data="Websocket connected.")
+
+    async def receive(self, text_data):
+        await self.send(text_data=f"You sent: {text_data}")
+
+    async def disconnect(self, code):
+        pass
```

Comment:
Currently, `disconnect` is a no-op. At minimum, log the disconnect code:
```
    async def disconnect(self, code):
        print(f"WebSocket disconnected with code: {code}")
```
This helps debug client disconnects or network failures.